cmds/tui.c

```diff
@@ -2401,6 +2410,20 @@ static void tui_main_loop(struct opts *opts, struct uftrace_data *handle)
 			cancel_search();
 			tui_window_move_down(win);
 			break;
+		case KEY_LEFT:
+			if (win == &graph->win || win == &partial_graph.win) {
+				if (leftcut >= 1) {
+					leftcut -= 1;
+					full_redraw = true;
+				}
+			}
+			break;
+		case KEY_RIGHT:
+			if (win == &graph->win || win == &partial_graph.win) {
+				leftcut += 1;
+				full_redraw = true;
+			}
+			break;
```

Comment:
So is this only for graph window?  I guess it should be easy (and useful) to apply to other windows too.

Reply:
Right, but I thought it might not be practically needed for other windows.